app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -176,23 +176,13 @@ fun PocketSponsoredStory(
 
         Spacer(Modifier.height(9.dp))
 
-        Row(verticalAlignment = Alignment.CenterVertically) {
-            Icon(
-                painter = painterResource(id = R.drawable.pocket_star_stroke),
-                contentDescription = null,
-                tint = FirefoxTheme.colors.iconSecondary,
-            )
-
-            Spacer(Modifier.width(8.dp))
-
-            Text(
-                text = stringResource(R.string.pocket_stories_sponsor_indication),
-                color = FirefoxTheme.colors.textSecondary,
-                fontSize = 12.sp,
-                overflow = TextOverflow.Ellipsis,
-                maxLines = 1,
-            )
-        }
+        Text(
+            text = stringResource(R.string.pocket_stories_sponsor_indication),
+            color = FirefoxTheme.colors.textSecondary,
+            fontSize = 12.sp,
```

Comment:
While we are making this change, can you check the figma for this feature and see if this fits under any of the FenixTypography and switch to the appropriate text style if any? My guess would be Caption without verifying with the figma, but we should double check.

Reply:
The design from https://www.figma.com/file/wsvGxcKQEKgI4zPSzB0Kk6/SPOC-Placement-on-Android-%26-iOS?node-id=4%3A9020 doesn't quite match the available styles. Asked on Jira for maybe updating the design.

Reply:
I think this is indeed using Caption when I inspected the node although it is using the iOS Caption given the SF Pro font. You can see Caption in the Typography section. I also compared the font size, weights, etc and I feel pretty confident we can unblock this change and move forward. Unfortunately, the Android designs are just a copy of the iOS components and the proper Android design tokens weren't used.

<img width="700" alt="Screen Shot 2022-08-24 at 11 47 34 AM" src="https://user-images.githubusercontent.com/1190888/186463599-3b38c7df-b0e7-4f17-b4a0-6f7edc752231.png">

Reply:
Thank you.
That text seem to match `Caption` but the other two texts for the title and the sponsor name do not currently match any of our styles. Waiting for @Verdi 's feedback about them in https://mozilla-hub.atlassian.net/browse/FNXV2-21199?focusedCommentId=580916